postman/collections/graviteeio-am-self-account-management-collection-app-version.json

```diff
@@ -1806,7 +1806,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n    \"factors\": [      \n       \"{{recoveryCodeFactorId}}\"\n    ],\n    \"settings\": {\n        \"mfa\": {\n            \"factor\": {\n                \"applicationFactors\": [ {\n                    \"id\": \"{{recoveryCodeFactorId}}\"\n                }]\n            }\n        }\n    }\n}"
+											"raw": "{\n    \"factors\": [      \n       \"{{recoveryCodeFactorId}}\"\n    ],\n    \"settings\": {\n        \"mfa\": {\n            \"factor\": {\n                \"applicationFactors\": [ {\n                    \"id\": \"{{recoveryCodeFactorId}}\"\n                }],\n                \"defaultFactorId\":\"{{recoveryCodeFactorId}}\"\n            }\n        }\n    }\n}"
```

Comment:
**NOTE**: we shouldn't be able to define a recoverycode as default factor, this is an issue to fix. Can you please confirm that point with Product and open a Private bug if I'm right.

Reply:
Yes, it is. I will create one